generators/vsix-vs-win/BotBuilderVSIX-V4/app.config

```diff
@@ -1,8 +1,7 @@
 <?xml version="1.0" encoding="utf-8"?>
 <configuration>
-  <runtime>
-  </runtime>
-  <startup>
-    <supportedRuntime version="v4.0" sku=".NETFramework,Version=v4.7"/>
-  </startup>
+	<runtime>
+	</runtime>
+	<startup>
+	</startup>
```

Comment:
I'm not sure what this change does. Why remove the previous supported Runtime attribute?

Reply:
I removed it because I thought it was not used. Just like my comment at https://github.com/Microsoft/BotBuilder-Samples/pull/1072/files/06b584b03f132a37357281dbb3f4729ebfb87b8f?utf8=%E2%9C%93&diff=split#r240155949